dev/ts/main.ts

```diff
@@ -105,6 +106,21 @@ export function getElementsByIds(bpmnId: string | string[]): BpmnElement[] {
   return bpmnVisualization.bpmnElementsRegistry.getElementsByIds(bpmnId);
 }
 
+function getParentElement(id: string): mxCell {
+  const cell = bpmnVisualization.graph.model.getCell(id);
+  return bpmnVisualization.graph.getModel().getParent(cell);
```

Comment:
**thought:** inconsistency. Calling model and getModel. Use a single form or introduce a model constant.